kadi/commands/states.py

```diff
@@ -1159,13 +1167,22 @@ def get_states(start=None, stop=None, state_keys=None, cmds=None, continuity=Non
     # reference the last state in the list.
     state = states[0]
 
+    # List of transitions that occur *after* the ``stop`` date but are needed for
```

Comment:
This comment would be less confusing to me if it mentioned that these transitions are all  from the commands (`cmds`) still in the continuity ranges (all before `stop`).  

`stop` is also a confusing name for me here (`date` -> `stop` and really it is just the reference time for continuity, not a `start` -> `stop` range, but that's neither here nor there).